src/Chapter2.hs

```diff
@@ -781,14 +859,24 @@ Let's now try to eta-reduce some of the functions and ensure that we
 mastered the skill of eta-reducing.
 -}
 divideTenBy :: Int -> Int
-divideTenBy x = div 10 x
+divideTenBy = div 10
 
 -- TODO: type ;)
+listElementsLessThan :: Int -> [Int] -> [Int]
 listElementsLessThan x l = filter (< x) l
 
--- Can you eta-reduce this one???
+-- Can you eta-reduce this one??? 
+{- NO 
+Why? I am guessing that if there are more than one parameters, the order of parameters matters.
+For example, zipWith (-) a b  VS zipWith (-) b a 
+-}
+pairMul :: [Int] -> [Int] -> [Int]
 pairMul xs ys = zipWith (*) xs ys
 
+{-
+TODO: Curious about what it means by 
+- provide more composable interfaces.
```

Comment:
Is your question applied to something specific, or you wonder what is composability generally?